package com.posadskiy.auth.core.exception;

public class UserWithProvidedEmailAlreadyExistException extends RuntimeException {}
